frontend/src/components/worksheets/NewRun/NewRun.jsx

```diff
@@ -241,6 +243,13 @@ class NewRun extends React.Component<{
         if (cpu) args.push(`--request-cpus=${cpu}`);
         if (gpu) args.push(`--request-gpus=${gpu}`);
         if (docker) args.push(`--request-docker-image=${docker}`);
+        if (queue) {
+            let finalQueue = queue;
+            if (finalQueue.startsWith("tag=")) {

```

Comment:
Why remove `tag=`? Given that we support both `--request-queue ...` and `--request-queue tag=...`

Reply:
Percy mentioned to strip it asap

Reply:
Why? It seems simpler just to say "what you type in is verbatim passed as the argument to `--request-queue`" @percyliang

Reply:
I agree it's good to be consistent in both places, but I think we should deprecate `tag=...` though.  The rationale for stripping `tag=` as early as possible was so most of our code can just reason about the non-stripped version, but open to other suggestions.

Reply:
I would go more for whatever is passed in is the tag if we support both. Unless we start rejecting  `tag=`, users who are used to it will continue to use that in command line as well and we won't be able to deprecate that